assignments/assignment01.rb

```diff
@@ -38,6 +38,7 @@ def number_to_string(n, lang)
 # it accepts a string
 # and returns the same string with each word capitalized.
 def titleize(s)
+  s.downcase.split.each { |w| w.capitalize! }.join " "
```

Comment:
Looks exactly right. You used a few features (such as each and join) that we had not yet covered in class.